.storybook/preview.js

```diff
@@ -82,5 +85,4 @@ const metamaskDecorator = (story, context) => {
   );
 };
 
-addDecorator(withKnobs);
 addDecorator(metamaskDecorator);
```

Comment:
Adding custom theme and removing duplicate adding of knobs addon

Reply:
Where else is this decorator added?

Reply:
[in`main.js`](https://github.com/MetaMask/metamask-extension/blob/develop/.storybook/main.js#L11)

Reply:
Can verify knobs still work in the [storybook build on the Box component](https://410908-42009758-gh.circle-artifacts.com/0/storybook/index.html?path=/story/ui-components-ui-box-box-stories-js--default-story)
<img width="902" alt="Screen Shot 2021-12-07 at 9 49 01 AM" src="https://user-images.githubusercontent.com/8112138/144920419-e445153e-f684-4c38-881c-cee107461796.png">

Reply:
That's odd. The docs suggest that you're supposed to add this to the list of addons *and* include the `withKnobs` decorator. Check out the examples: https://github.com/storybookjs/addon-knobs#readme They all use the decorator.

Reply:
I would tentatively suggest that the docs haven't been updated as [addDecorator has been deprecated in v6](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated-addparameters-and-adddecorator) knobs as a whole have been deprecated in favour or controls so I would be looking to convert all our stories as well. It is [also mentioned here](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#new-addon-presets)

<img width="1332" alt="Screen Shot 2021-12-07 at 12 22 29 PM" src="https://user-images.githubusercontent.com/8112138/144938733-5d621b31-a6d0-4eb7-8f84-168f5f560d35.png">

Reply:
Ah, great find, thanks! I didn't see that.

Reply:
Thanks for the thorough review 🙏